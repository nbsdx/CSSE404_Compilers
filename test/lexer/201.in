/** Testing basic comments and
  * uncomplicated, well-formed program.
  */

class Trivial {
    public static void main (String[] args) {
        boolean taut = true;
        int declared_int = 12345;

        while (taut) {
            declared_int = declared_int + 1;
        }

        return 0;
    }
}
