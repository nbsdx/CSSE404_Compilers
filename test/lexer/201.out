ReservedWord, class
ID, Trivial
Delimiter, {
ReservedWord, public
ReservedWord, static
ReservedWord, void
ReservedWord, main
Delimiter, (
ReservedWord, String
Delimiter, [
Delimiter, ]
ID, args
Delimiter, )
Delimiter, {
ReservedWord, boolean
ID, taut
Delimiter, =
ReservedWord, true
Delimiter, ;
ReservedWord, int
ID, declared_int
Delimiter, =
Integer, 12345
Delimiter, ;
ReservedWord, while
Delimiter, (
ID, taut
Delimiter, )
Delimiter {
ID, declared_int
Delimiter, =
ID, declared_int
Operator, +
Integer, 1
Delimiter, }
ReservedWord, return
Integer, 0
Delimiter, }
Delimiter, }
